backend/src/main/java/com/bakdata/conquery/apiv1/query/TableExportQuery.java

```diff
@@ -206,18 +205,17 @@ private static Map<ColumnId, Integer> calculateColumnPositions(
 						continue;
 					}
 
-					final ColumnId columnId = column.getId();
-					if (positions.containsKey(columnId)) {
+					final ColumnId id = column.getId();
+					if (positions.containsKey(id)) {
 						continue;
 					}
 
-					// We want to have ConceptColumns separate here.
-					if (column.getSecondaryId() != null && !conceptColumns.contains(column.getId())) {
-						positions.putIfAbsent(columnId, secondaryIdPositions.get(column.getSecondaryId()));
```

Comment:
Angenommen eine column ist secondary Id und ebenfalls concept column und angenommen, die berechnete Position ist 4 und current Position ist 7, dann wird die Column fälschlicherweise nicht an Position 4 priorisiert eingetragen, weil secondary Id, sondern wird bei Position 7 eingesetzt, weil ebenfalls in concept columns.